### PR TITLE
Update repair oriented examples

### LINKED ISSUE #3557: docs: Issue in page Repair
I would like to report an issue in page https://manager.docs.scylladb.com/branch-3.2/sctool/repair

### Problem
I found it challenging to understand whether and what intensity and parallelism settings via scylla manager commands.

###  Suggest a fix
It would have saved me a lot of time if In the section discussing the intensity and parallelism settings, it was indicated when the properties are set, they are visible in sctool tasks --show-properties setting.  If these defaults are used, indicate --show-properties would display nothing.

### LINKED ISSUE #3562: docs: Issue in page Repair
I would like to report an issue in page https://manager.docs.scylladb.com/branch-3.0/sctool/repair

### Problem
The --enabled flag takes --enabled=false.  It is not clear from the documentation, and people struggle with getting it to successfully work.

###  Suggest a fix
Please include an example disabling a scheduled repair, displaying the disabled repair in the task list , and re-enabling the disabled repair using repair id.

sctool tasks -a --show-ids
Weekly repair task is on list: repair/54c2432d-20e2-4d18-ae62-c2ab227579a6 

sctool repair update -c my-cluster-id repair/54c2432d-20e2-4d18-ae62-c2ab227579a6 --enabled=false

# showing tasks with -a is required to see disabled repair.  
sctool tasks -a --show-ids
Disabled weekly repair task can be seen with leading asterick: *repair/54c2432d-20e2-4d18-ae62-c2ab227579a6 

sctool repair update -c my-cluster-id repair/54c2432d-20e2-4d18-ae62-c2ab227579a6 --enabled=true

Then confirm the repair is re-enabled by once again displaying tasks.  The asterick should be removed.  (The -a option is not required, if the task remains disabled, it will not be visible.)
sctool tasks --show-ids
re-enabled weekly task returns to the list, no asterick prefix: repair/54c2432d-20e2-4d18-ae62-c2ab227579a6

### PR DESCRIPTION
**Pull request by Michal-Leszczynski**
Fixes #3557
Fixes #3562

### CODE REVIEW

**Review by karol-kokoszka — APPROVED** [review]

**Comment by Michal-Leszczynski** [comment]
Hi @annastuchlik, could you take a look at those changes?

File: docs/source/repair/examples.rst [hunk 1]
```diff
@@ -153,6 +153,29 @@ Monitor progress of the repair task
...
+  ├───────────────────────────────┼────────────────────────────────┼──────────┼──────────┤
+  │ system_auth                   │                role_attributes │ 100%     │ 0s       │
+  │ system_auth                   │                   role_members │ 100%     │ 0s       │
+  │ system_auth                   │               role_permissions │ 0%       │ 0s       │
+  │ system_auth                   │                          roles │ 0%       │ 0s       │
+  ╰───────────────────────────────┴────────────────────────────────┴──────────┴──────────╯
+
+Note that the ``Parallel`` and ``Intensity`` properties are displayed only for a running repair task and include
+changes applied via :ref:`sctool repair control <repair-control>`.
+For checking parameters of not running repair task use :ref:`sctool tasks --show-properties <task-list>`.
```
Comment (annastuchlik) [inline docs/source/repair/examples.rst:181]:
```suggestion
To check the parameters of not running repair tasks, use :ref:`sctool tasks --show-properties <task-list>`.
```

File: docs/source/sctool/partials/sctool_repair.yaml [hunk 2]
```diff
@@ -60,6 +60,8 @@ options:
     How many token ranges to repair in a single Scylla node at the same time.
     Zero (0) is a special value, the number of token ranges is adjusted to the maximum supported by node (see max_repair_ranges_in_parallel in Scylla logs).
     Changing the intensity impacts repair granularity if you need to resume it, the higher the value the more work on resume.
+    Setting intensity to a value greater than max supported results in trimming it internally.
```
Comment (annastuchlik) [inline docs/source/sctool/partials/sctool_repair.yaml:63]: This sentence is confusing to me. It looks like it's about "max supported results". And I don't understand what "internally" means here. Perhaps it should go like this:

_If you set intensity to a value greater than the maximum supported by the node, intensity will be capped at that maximum._

File: docs/source/sctool/partials/sctool_repair.yaml [hunk 3]
```diff
@@ -108,6 +110,8 @@ options:
     The effective parallelism depends on a keyspace replication factor (RF) and the number of nodes.
     The formula to calculate is is as follows: number of nodes / RF.
     For example, the maximum parallelism for a 6 node cluster with RF=3 is 2.
+    Setting parallel to a value greater than max supported results in trimming it internally.
```
Comment (annastuchlik) [inline docs/source/sctool/partials/sctool_repair.yaml:113]: See my comment about `intensity`. If my suggestion makes sense to you, you can apply it here as well.

File: docs/source/sctool/partials/sctool_repair_update.yaml [hunk 4]
```diff
@@ -59,6 +61,8 @@ options:
     How many token ranges to repair in a single Scylla node at the same time.
     Zero (0) is a special value, the number of token ranges is adjusted to the maximum supported by node (see max_repair_ranges_in_parallel in Scylla logs).
     Changing the intensity impacts repair granularity if you need to resume it, the higher the value the more work on resume.
+    Setting intensity to a value greater than max supported results in trimming it internally.
```
Comment (annastuchlik) [inline docs/source/sctool/partials/sctool_repair_update.yaml:64]: See my comment about `intensity`. If my suggestion makes sense to you, you can apply it here as well.

File: docs/source/sctool/partials/sctool_repair_update.yaml [hunk 5]
```diff
@@ -107,6 +111,8 @@ options:
     The effective parallelism depends on a keyspace replication factor (RF) and the number of nodes.
     The formula to calculate is is as follows: number of nodes / RF.
     For example, the maximum parallelism for a 6 node cluster with RF=3 is 2.
+    Setting parallel to a value greater than max supported results in trimming it internally.
```
Comment (annastuchlik) [inline docs/source/sctool/partials/sctool_repair_update.yaml:114]: See my comment about `intensity`. If my suggestion makes sense to you, you can apply it here as well.

File: pkg/command/repair/res.yaml [hunk 6]
```diff
@@ -19,13 +19,17 @@ intensity: |
   How many token ranges to repair in a single Scylla node at the same time.
   Zero (0) is a special value, the number of token ranges is adjusted to the maximum supported by node (see max_repair_ranges_in_parallel in Scylla logs).
   Changing the intensity impacts repair granularity if you need to resume it, the higher the value the more work on resume.
+  Setting intensity to a value greater than max supported results in trimming it internally.
```
Comment (annastuchlik) [inline pkg/command/repair/res.yaml:22]: See my comment about `intensity`. If my suggestion makes sense to you, you can apply it here as well.

File: pkg/command/repair/res.yaml [hunk 7]
```diff
@@ -19,13 +19,17 @@ intensity: |
...
+  See effectively used intensity value in the display of 'sctool progress repair' command.
 
 parallel: |
   The maximum number of Scylla repair jobs that can run at the same time (on different token ranges and replicas).
   Each node can take part in at most one repair at any given moment. By default the maximum possible parallelism is used.
   The effective parallelism depends on a keyspace replication factor (RF) and the number of nodes.
   The formula to calculate is is as follows: number of nodes / RF.
   For example, the maximum parallelism for a 6 node cluster with RF=3 is 2.
+  Setting parallel to a value greater than max supported results in trimming it internally.
```
Comment (annastuchlik) [inline pkg/command/repair/res.yaml:31]: See my comment about `intensity`. If my suggestion makes sense to you, you can apply it here as well.

File: docs/source/sctool/partials/sctool_tasks.yaml [hunk 8]
```diff
@@ -27,6 +27,7 @@ options:
   default_value: "false"
   usage: |
     Additionally display task properties.
+    Only set properties are displayed, the rest of them are set to their default values.
```
Comment (annastuchlik) [inline docs/source/sctool/partials/sctool_tasks.yaml:30]: Perhaps:
_Only displays properties set to non-default values._

File: pkg/command/tasks/res.yaml [hunk 9]
```diff
@@ -16,6 +16,7 @@ show-ids: |
 
 show-properties: |
   Additionally display task properties.
+  Only set properties are displayed, the rest of them are set to their default values.
```
Comment (annastuchlik) [inline pkg/command/tasks/res.yaml:19]: Perhaps:
_Only displays properties set to non-default values._

**Comment by annastuchlik** [comment]
@Michal-Leszczynski I've left some comments. See if they make sense to you.

**Pull request merged**